src/Main.java

```diff
@@ -1,5 +1,105 @@
+import java.util.*;
+
 public class Main {
+    public static int countOccurrence(List<String> words, String target) {
```

Comment:
not necessarily, but consider using null checks, e.g. validate check if words is not null and not empty